src/js/Manager/AutofillManager.js

```diff
@@ -27,13 +30,24 @@ export default new class AutofillManager {
     async _sendAutofillPassword(recommendations) {
         if(recommendations.length === 0 || await SettingsService.getValue('paste.autofill') === false) return;
         let password = recommendations[0];
+        var self = this;
 
-        let ids = TabManager.get('autofill.ids', []);
-        if(ids.indexOf(password.getId()) === -1) {
-            ids.push(password.getId());
-            TabManager.set('autofill.ids', ids);
-        }
+        setTimeout(function () {
```

Comment:
You can actually use an arrow function here. This solves the problem with `this`.
```suggestion
        setTimeout(function () {
        setTimeout(() => {
```

Reply:
Fixed